spaceship/lib/spaceship/portal/certificate.rb

```diff
@@ -241,29 +270,46 @@ def factory(attrs)
           klass.new(attrs)
         end
 
+        # <b>DEPRECATED:</b> Use <tt>all_by_platform</tt> instead.
         # @param mac [Bool] Fetches Mac certificates if true. (Ignored if callsed from a subclass)
         # @return (Array) Returns all certificates of this account.
         #  If this is called from a subclass of Certificate, this will
         #  only include certificates matching the current type.
         def all(mac: false)
+          puts '`all` is deprecated. Please use `all_by_platform` instead.'.red
+          all_by_platform(platform: mac ? Spaceship::Portal::App::MAC : Spaceship::Portal::App::IOS)
+        end
+
+        # @param platform [String] Fetches platform certificates. (Ignored if called from a subclass)
+        # @return (Array) Returns all certificates of this account.
+        #  If this is called from a subclass of Certificate, this will
+        #  only include certificates matching the current type.
+        def all_by_platform(platform: Spaceship::Portal::App::IOS)
```

Comment:
@samrobbins probably a branch from mine so you can easily compare to see if something is missing? How would that then work with the existing all (mac: false), etc. Would those still become deprecated or would they just drop the param?